templates/rhsm-subscriptions-scheduler.yml

```diff
@@ -713,3 +715,122 @@ objects:
                     secretName: splunk
                 - name: logs
                   emptyDir:
+  - apiVersion: batch/v1beta1
+    kind: CronJob
+    metadata:
+      name: offering-sync
+    spec:
+      schedule: ${OFFERING_SYNC_SCHEDULE}
+      jobTemplate:
+        spec:
+          activeDeadlineSeconds: 1800
+          template:
+            spec:
+              activeDeadlineSeconds: 1800
+              restartPolicy: Never
+              imagePullSecrets:
+                - name: ${IMAGE_PULL_SECRET}
+                - name: quay-cloudservices-pull
+              containers:
+                - image: ${IMAGE}:${IMAGE_TAG}
+                  name: offering-sync
+                  env:
+                    - name: SPRING_PROFILES_ACTIVE
+                      value: offering-sync,kafka-queue
+                    - name: JAVA_MAX_MEM_RATIO
+                      value: '85'
+                    - name: GC_MAX_METASPACE_SIZE
+                      value: '256'
+                    - name: LOG_FILE
+                      value: /logs/server.log
+                    - name: LOGGING_LEVEL_ROOT
+                      value: ${LOGGING_LEVEL_ROOT}
+                    - name: LOGGING_LEVEL_ORG_CANDLEPIN
+                      value: ${LOGGING_LEVEL}
+                    - name: KAFKA_BOOTSTRAP_HOST
+                      value: ${KAFKA_BOOTSTRAP_HOST}
+                    - name: DATABASE_HOST
+                      valueFrom:
+                        secretKeyRef:
+                          name: rhsm-db
+                          key: db.host
+                    - name: DATABASE_PORT
+                      valueFrom:
+                        secretKeyRef:
+                          name: rhsm-db
+                          key: db.port
+                    - name: DATABASE_USERNAME
+                      valueFrom:
+                        secretKeyRef:
+                          name: rhsm-db
+                          key: db.user
+                    - name: DATABASE_PASSWORD
+                      valueFrom:
+                        secretKeyRef:
+                          name: rhsm-db
+                          key: db.password
+                    - name: DATABASE_DATABASE
+                      valueFrom:
+                        secretKeyRef:
+                          name: rhsm-db
+                          key: db.name
+                    - name: INVENTORY_DATABASE_HOST
+                      valueFrom:
+                        secretKeyRef:
+                          name: host-inventory-db-readonly
+                          key: db.host
+                    - name: INVENTORY_DATABASE_DATABASE
+                      valueFrom:
```

Comment:
Doesn't seem like these INVENTORY_* variables are required for this job, or are they? Could this be copy/paste issues?

Reply:
That's a good question, I don't know if they're required or not. This was copied nearly verbatim from rhsm-subscriptions-cron-purge since it seemed close enough to the cron that this needs to have. If you'd like I can tear it out.

Reply:
> That's a good question, I don't know if they're required or not. This was copied nearly verbatim from rhsm-subscriptions-cron-purge since it seemed close enough to the cron that this needs to have. If you'd like I can tear it out.

If the subscription sync code doesn't need to look at the inventory database, then I don't think it is required. I'm not close enough to that code to know for sure, but to me it doesn't seem like the two would be related.

@kahowell Any idea if the INVENTORY_* env vars are required here?

Reply:
The INVENTORY_* env vars shouldn't be required here. Please remove them; however, if you find this breaks the deployment, I'm happy to punt for now and fix with a tech-debt card later. (We have other deployments that have these vars when they don't need to).

Reply:
Sounds good! Deleted.